indy_node/test/upgrade/test_pool_upgrade_force_scheduled_only_once.py

```diff
@@ -0,0 +1,9 @@
+from indy_node.server.upgrade_log import UpgradeLog
+
+
+def testPoolUpgradeForceScheduledOnlyOnce(upgradeScheduledExpForceTrue,
```

Comment:
Do we have a test that schedules update twice and checks that they work as expected (as in INDY-382)?

Reply:
Added the tests `test_node_schedules_upgrade_for_proper_datetime` and `test_node_reschedules_upgrade_for_proper_datetime` for verifying actual schedule of pool upgrade for the cases when POOL_UPGRADE transaction is sent once and twice.